src/api.js

```diff
@@ -17,3 +17,17 @@ export const getReviews = (category) => {
     });
   }
 };
+
+export const getReviewById = (review_id) => {
+  return gamesApi.get(`/reviews/${review_id}`).then(({ data }) => {
+      const { review } = data; 
+      return review; 
```

Comment:
Dont know if you're interested, but you can destructure inside a destructuring i.e.

` return gamesApi.get(`/reviews/${review_id}`).then(({ data: {review} }) => {
      return review; `
      
      key bit is: `{ data: {review} }`